values-cs/strings.xml

```diff
@@ -2,126 +2,126 @@
-    <string name="failed_to_take_screenshot">Failed to take screenshot</string>
+    <string name="failed_to_take_screenshot">Pří ukládání došlo k chybě</string>
     <!-- Toast message that shows when there was a problem making an in-app purchase -->
-    <string name="failed_to_complete_purchase">Failed to complete purchase</string>
+    <string name="failed_to_complete_purchase">Při nákupu došlo k chybě</string>
 
     <!-- MISC STRINGS -->
-    <string name="play_games_sign_in_failed">Failed to sign in. Please check your network connection and try again.</string>
-    <string name="unknown_error">Unknown error</string>
-    <string name="unknown_error_please_try_again_later">Unknown error. Please try again later.</string>
-    <string name="buy_all_games_purchase_description">Buy all games, including any new games we release in the future!</string>
+    <string name="play_games_sign_in_failed">Při přihlašování došlo k chybě. Zkontrolujte internetové připojení a zkuste to znovu.</string>
+    <string name="unknown_error">Neznámá chyba</string>
+    <string name="unknown_error_please_try_again_later">Neznámá chyba. Prosím zkuste to znovu později.</string>
+    <string name="buy_all_games_purchase_description">Koupě všech her zahrnuje i hry přidané v budoucnosti!</string>
 
     <!-- WHAT'S NEW DIALOG STRINGS -->
-    <string name="rate_app">Rate app</string>
-    <string name="whats_new">What\'s new</string>
+    <string name="rate_app">Ohodnoť aplikaci</string>
+    <string name="whats_new">Co je nového?</string>
     <!-- '%1$s' will be replaced with the version number (e.g. '1.15') -->
-    <string name="version_number_format">Version %1$s</string>
+    <string name="version_number_format">Verze %1$s</string>
     <string-array name="whats_new_features_v12">
-        <item>New game: Hex!</item>
-        <item>16384 is now free!</item>
-        <item>Smoother, slicker animations</item>
-        <item>Bug fixes &amp; enhancements</item>
+        <item>Nová hra: Hex!</item>
+        <item>16384 je nyní zdarma!</item>
+        <item>Nové animace</item>
+        <item>Opraveny chyby &amp; vylepšení</item>
     </string-array>
-    <string name="enjoy">Enjoy!</string>
+    <string name="enjoy">Užijte si to!</string>
 
     <!-- HOW TO PLAY HEX STRINGS -->
-    <string name="how_to_play_hex_paragraph1">Swipe left, right, or diagonally to slide the tiles across the grid. When <b>three 1-tiles</b> or any other <b>two tiles</b> with same number touch, they merge into one.</string>
-    <string name="how_to_play_hex_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move.</string>
+    <string name="how_to_play_hex_paragraph1">Přejeď doleva, doprava nebo diagonálně k posunutí dlaždice po ploše.3 dlaždice s číslem 1 nebo jakékoliv 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
```

Comment:
Should there be a space in between ".3" since it is the start of a new sentence?